plugins/chat/config/locales/server.en.yml

```diff
@@ -176,6 +176,8 @@ en:
       inline_to_message: "Message #%{message_id} by %{username} – #%{chat_channel}"
       inline_to_channel: "Chat #%{chat_channel}"
       inline_to_topic_channel: "Chat for Topic %{topic_title}"
+      default_thread_title: "Thread"
+      thread_title_connector: "in"
```

Comment:
This doesn't seem right to me.
```
thread_title_connector: I18n.t("chat.onebox.thread_title_connector"),
```


```
<span class="thread-title-connector">{{{thread_title_connector}}}</span>
```
Is there a better way to do this?

Reply:
We don't use default thread title anymore IMO, what do you say @chapoi ?

Reply:
Yes, just leave it blank. I have an upcoming design idea for it anyway.

I do think the images are looking very big, might be the perspective of the screenshots, but might need to be scaled down a little. Will be more clear when it's in live, so don't let that block.

Reply:
Yes agree on the size of images @jancernik. Maybe we could have some gallery like solution?

![Screenshot 2023-09-26 at 14 06 06](https://github.com/discourse/discourse/assets/339945/23346345-5f6b-4f65-bfeb-ba6d75d54860)

Something like this?